### PR TITLE
Text proposal for this year

### PR DESCRIPTION
**Pull request by pplu**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -45,8 +45,20 @@
...
+								<h2>A free one-day "conference" for Geeks and Friends</h2>
+								<p>
+                  The computer industry is becoming a place where you are trapped 
+                  in the feeling that you're missing out if you're not using last weeks 
+                  "hot thing" commented on twitter, and that no one will be interested if 
+                  you're not talking about the last functional programming lanaguage, new 
+                  framework, etc. The fact is that innovations and amazing things are going 
+                  on in lots of our "out of sight" communities. Come here to show us what 
+                  you've done, how you like to work, what is working for you and what is not, 
+                  without the feeling that you'll be frowned upon because you say "Jquery",
```
Comment (meis) [inline index.html]: Nitpick; JQuery?

Comment (pplu) [reply index.html]: +1. Sorry for the imprecision

Comment (pplu) [reply index.html]: BTW: wouldn't you bold those names, instead of quoting them? Maybe its less noisy

**Comment by meis** [comment]
Already merged on #2

**Pull request closed without merging**